movebucket_test.go

```diff
@@ -143,7 +143,7 @@ func TestTx_MoveBucket(t *testing.T) {
 	for _, tc := range testCases {
 
 		t.Run(tc.name, func(*testing.T) {
-			db := btesting.MustCreateDBWithOption(t, &bbolt.Options{PageSize: 4096})
+			db := btesting.MustCreateDBWithOption(t, &bbolt.Options{PageSize: pageSize})
```

Comment:
Minor comment:
Actually I prefer not to depend on a variable in another test file. It's exactly the reason why I changed it to 4096 previously.

This comment applies to all the following places which use the variable `pageSize`. Suggest to either use 4096 directly or define a local variable instead.

Reply:
done